tool/tsh/tsh.go

```diff
@@ -4328,9 +4328,6 @@ func reissueWithRequests(cf *CLIConf, tc *client.TeleportClient, newRequests []s
 	if err != nil {
 		return trace.Wrap(err)
 	}
-	if profile.IsVirtual {
-		return trace.BadParameter("cannot reissue certificates while using an identity file (-i)")
-	}
```

Comment:
Now that[ identity files are treated as an in memory store](https://github.com/gravitational/teleport/pull/19420), this restriction is no longer necessary. The user's certificates will just be updated in memory to carry out that single request. This also applies to `--add-keys-to-agent=only` and `--headless`.